Source/ui_qt/Qt_ui/settingsdialog.ui

```diff
@@ -260,6 +260,41 @@
         <string>Enable Audio Output</string>
        </property>
       </widget>
+      <widget class="QSpinBox" name="spinBox_spuBlockCount">
+       <property name="geometry">
+        <rect>
+         <x>20</x>
+         <y>70</y>
+         <width>121</width>
+         <height>22</height>
+        </rect>
+       </property>
+       <property name="minimum">
+        <number>10</number>
+       </property>
+       <property name="maximum">
+        <number>400</number>
+       </property>
+       <property name="singleStep">
+        <number>10</number>
+       </property>
+       <property name="value">
+        <number>400</number>
+       </property>
+      </widget>
+      <widget class="QLabel" name="label_4">
+       <property name="geometry">
+        <rect>
+         <x>20</x>
+         <y>50</y>
+         <width>171</width>
+         <height>16</height>
+        </rect>
+       </property>
+       <property name="text">
+        <string>SPU Block Count*:</string>
```

Comment:
Why is there an asterisk here? Is it a typo?

Reply:
I was going to add an explanation to it, but I wasn't sure what to write.

If you have a suggestion, else I'll remove it